cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingStepPage {
+    private acceptTerms: string;
+
+    private proceedToPayment: string;
+
+    constructor() {
+      this.acceptTerms = ".checkbox span";
+      this.proceedToPayment = ".cart_navigation span";
+    }
+
+    public clickTerms():void {
+      cy.get(this.acceptTerms).click();
+    }
+
+    public proceedToNextPage(): void {
```

Comment:
instead of next page you can use the name of the next page, proceedToPayment, or proceedToCheckout also can work